package.json

```diff
@@ -12,7 +12,8 @@
     "husky": "^8.0.0",
     "lint-staged": "^12.4.3",
     "prettier": "^2.8.8",
-    "tsx": "^4.7.1"
+    "tsx": "^4.7.1",
+    "zksync-ethers": "^5"
```

Comment:
only tooling shared across the monorepo should be in this root package.json. Move it to the individual pacakges and specify a more specific version number

Reply:
We pretty much use it across the monorepo (solidity/sdk/cli), will specify version number